---
title: An integrated view of the structure and function of the human 4D nucleome
image: dekker-2025-598VEH85.png
image-alt: >-
  Graphical abstract. Overview of 4D Nucleome project, integrating chromatin interaction mapping, nuclear positioning,
  imaging, single-cell Hi-C, and structure modeling to analyze 3D genome organization and function.
members:
  - nils-gehlenborg
year: 2025
type: article
publisher: 'https://www.nature.com/articles/s41586-025-09890-3'
doi: 10.1038/s41586-025-09890-3
cite:
  authors: >-
    J Dekker, BA Oksuz, Y Zhang, Y Wang, MK Minsk, S Kuang, L Yang, JH Gibcus, N Krietenstein, OJ Rando, J Xu, DH
    Janssens, S Henikoff, A Kukalev, W Andréa, W Winick-Ng, R Kempfer, A Pombo, M Yu, P Kumar, L Zhang, AS Belmont, T
    Sasaki, T Van Schaik, L Brueckner, D Peric-Hupkes, B Van Steensel, P Wang, H Chai, M Kim, Y Ruan, R Zhang, SA
    Quinodoz, P Bhat, M Guttman, W Zhao, S Chien, Y Liu, SV Venev, D Plewczynski, II Azcarate, D Szabó, CJ Thieme, T
    Szczepińska, M Chiliński, K Sengupta, M Conte, A Esposito, A Abraham, R Zhang, Y Wang, X Wen, Q Wu, Y Yang, J Liu, L
    Boninsegna, A Yildirim, Y Zhan, AM Chiariello, S Bianco, L Lee, M Hu, Y Li, RJ Barnett, AL Cook, DJ Emerson, C
    Marchal, P Zhao, PJ Park, BH Alver, AJ Schroeder, R Navelkar, C Bakker, W Ronchetti, S Ehmsen, AD Veit, N
    Gehlenborg, T Wang, D Li, X Wang, M Nicodemi, B Ren, S Zhong, JE Phillips-Cremins, DM Gilbert, KS Pollard, F Alber,
    J Ma, WS Noble, F Yue
  published: '*Nature*'
zotero-key: 598VEH85
videos: []
other-resources: []
awards: []
preprint: 'http://biorxiv.org/lookup/doi/10.1101/2024.09.17.613111'
---
The dynamic three-dimensional (3D) organization of the human genome (the 4D nucleome) is linked to genome function. Here we describe efforts by the 4D Nucleome Project 
              1 
              to map and analyse the 4D nucleome in widely used H1 human embryonic stem cells and immortalized fibroblasts (HFFc6). We produced and integrated diverse genomic datasets of the 4D nucleome, each contributing unique observations, which enabled us to assemble extensive catalogues of more than 140,000 looping interactions per cell type, to generate detailed classifications and annotations of chromosomal domain types and their subnuclear positions, and to obtain single-cell 3D models of the nuclear environment of all genes including their long-range interactions with distal elements. Through extensive benchmarking, we describe the unique strengths of different genomic assays for studying the 4D nucleome, providing guidelines for future studies. Three-dimensional models of population-based and individual cell-to-cell variation in genome structure showed connections between chromosome folding, nuclear organization, chromatin looping, gene transcription and DNA replication. Finally, we demonstrate the use of computational methods to predict genome folding from DNA sequence, which will facilitate the discovery of potential effects of genetic variants, including variants associated with disease, on genome structure and function.
