---
title: Scalable Open Science Approach for Mutation Calling of Tumor Exomes Using Multiple Genomic Pipelines
image: ellrott-2018-QRCZCI5S.png
image-alt: Graphical Abstract
members:
  - nils-gehlenborg
year: 2018
type: article
publisher: 'https://linkinghub.elsevier.com/retrieve/pii/S2405471218300966'
doi: 10.1016/j.cels.2018.03.002
cite:
  authors: "*The Cancer Genome Atlas Research Network* (incl. N Gehlenborg)"
  published: '*Cell Systems* **6**(3):271-281.e7'
zotero-key: QRCZCI5S
videos: []
other-resources: []
awards: []
---
The Cancer Genome Atlas (TCGA) cancer genomics dataset includes over 10,000 tumor-normal exome pairs across 33 different cancer types, in total >400 TB of raw data files requiring analysis. Here we describe the Multi-Center Mutation Calling in Multiple Cancers project, our effort to generate a comprehensive encyclopedia of somatic mutation calls for the TCGA data to enable robust cross-tumor-type analyses. Our approach accounts for variance and batch effects introduced by the rapid advancement of DNA extraction, hybridization-capture, sequencing, and analysis methods over time. We present best practices for applying an ensemble of seven mutation-calling algorithms with scoring and artifact filtering. The dataset created by this analysis includes 3.5 million somatic variants and forms the basis for PanCan Atlas papers. The results have been made available to the research community along with the methods used to generate them. This project is the result of collaboration from a number of institutes and demonstrates how team science drives extremely large genomics projects.
