---
title: "Comprehensive and Integrative Genomic Characterization of Diffuse Lower Grade Gliomas"
image: characterize-gliomas.png
image-alt: A genomic data visualization that provides a comprehensive overview of the genomic landscape of glioma tumors involving mutation counts, patient characteristics like age and sex, as well as the molecular subtypes and mutation types observed across the samples.

members:
  - nils-gehlenborg
type: article
year: 2015
publisher: "http://www.nejm.org/doi/10.1056/NEJMoa1402121"
doi: "10.1056/NEJMoa1402121"
zotero-key: "VDBH3CLD"
cite:
  authors: "*The Cancer Genome Atlas Research Network* (incl. N Gehlenborg)"
  published: "*New England Journal of Medicine* **372**(26):2481-2498"
---
### Background
Diffuse low-grade and intermediate-grade gliomas (which together make up the lower-grade gliomas, World Health Organization grades II and III) have highly variable clinical behavior that is not adequately predicted on the basis of histologic class. Some are indolent; others quickly progress to glioblastoma. The uncertainty is compounded by interobserver variability in histologic diagnosis. Mutations in IDH, TP53, and ATRX and codeletion of chromosome arms 1p and 19q (1p/19q codeletion) have been implicated as clinically relevant markers of lower-grade gliomas.

### Methods
We performed genomewide analyses of 293 lower-grade gliomas from adults, incorporating exome sequence, DNA copy number, DNA methylation, messenger RNA expression, microRNA expression, and targeted protein expression. These data were integrated and tested for correlation with clinical outcomes.

### Results
Unsupervised clustering of mutations and data from RNA, DNA-copy-number, and DNA-methylation platforms uncovered concordant classification of three robust, nonoverlapping, prognostically significant subtypes of lower-grade glioma that were captured more accurately by IDH, 1p/19q, and TP53 status than by histologic class. Patients who had lower-grade gliomas with an IDH mutation and 1p/19q codeletion had the most favorable clinical outcomes. Their gliomas harbored mutations in CIC, FUBP1, NOTCH1, and the TERT promoter. Nearly all lower-grade gliomas with IDH mutations and no 1p/19q codeletion had mutations in TP53 (94%) and ATRX inactivation (86%). The large majority of lower-grade gliomas without an IDH mutation had genomic aberrations and clinical behavior strikingly similar to those found in primary glioblastoma.

### Conclusions
The integration of genomewide data from multiple platforms delineated three molecular classes of lower-grade gliomas that were more concordant with IDH, 1p/19q, and TP53 status than with histologic class. Lower-grade gliomas with an IDH mutation either had 1p/19q codeletion or carried a TP53 mutation. Most lower-grade gliomas without an IDH mutation were molecularly and clinically similar to glioblastoma. (Funded by the National Institutes of Health.)
