---
title: 'Comprehensive Analysis of Alternative Splicing Across Tumors from 8,705 Patients'
image: kahles-2018-tgca-alternative-splicing.png
image-alt: >-
  Graphical abstract. RNA splicing analysis in cancer, highlighting tumor-specific neojunctions, somatic alterations,
  MHC binding, and predicted neoepitopes
members:
  - nils-gehlenborg
year: 2018
type: article
publisher: 'https://linkinghub.elsevier.com/retrieve/pii/S1535610818303064'
doi: 10.1016/j.ccell.2018.07.001
cite:
  authors: "*The Cancer Genome Atlas Research Network* (incl. N Gehlenborg)"
  published: '*Cancer Cell* **34**(2):211-224.e6'
zotero-key: 8ZHR7LZH
videos: []
other-resources: []
awards: []
---
Our comprehensive analysis of alternative splicing across 32 The Cancer Genome
Atlas cancer types from 8,705 patients detects alternative splicing events and
tumor variants by reanalyzing RNA and whole-exome sequencing data. Tumors have
up to 30% more alternative splicing events than normal samples. Association
analysis of somatic variants with alternative splicing events confirmed known
trans associations with variants in SF3B1 and U2AF1 and identified additional
trans-acting variants (e.g., TADA1, PPP2R1A). Many tumors have thousands of
alternative splicing events not detectable in normal samples; on average, we
identified ≈930 exon-exon junctions ("neojunctions") in tumors not typically
found in GTEx normals. From Clinical Proteomic Tumor Analysis Consortium data
available for breast and ovarian tumor samples, we confirmed ≈1.7 neojunction-
and ≈0.6 single nucleotide variant-derived peptides per tumor sample that are
also predicted major histocompatibility complex-I binders (“putative
neoantigens”).
