---
title: "Cistrome Data Browser: integrated search, analysis and visualization of chromatin data"
image: cistrome-data-browser.png

members:
  - sehi-lyi
  - nils-gehlenborg

year: 2024
type: article

publisher: "https://doi.org/10.1093/nar/gkad1069"
cite:
  authors: "L Taing, A Dandawate, S L'Yi, N Gehlenborg, M Brown, C A Meyer"
  published: "*Nucleic Acids Research* **52**(D1), D61-D66"

---
The Cistrome Data Browser is a resource of ChIP-seq, ATAC-seq and DNase-seq data from humans and mice. It provides maps of the genome-wide locations of transcription factors, cofactors, chromatin remodelers, histone post-translational modifications and regions of chromatin accessible to endonuclease activity. Cistrome DB v3.0 contains approximately 45,000 human and 44,000 mouse samples with about 32,000 newly collected datasets compared to the previous release. The Cistrome DB v3.0 user interface is implemented as a single page application that unifies menu driven and data driven search functions and provides an embedded genome browser, which allows users to find and visualize data more effectively. Users can find informative chromatin profiles through keyword, menu, and data-driven search tools. Browser search functions can predict the regulators of query genes as well as the cell type and factor dependent functionality of potential cis-regulatory elements. Cistrome DB v3.0 expands the display of quality control statistics, incorporates sequence logos into motif enrichment displays and includes more expansive sample metadata. Cistrome DB v3.0 is available at http://db3.cistrome.org/browser.