---
title: Machine Learning Detects Pan-cancer Ras Pathway Activation in The Cancer Genome Atlas
image: way-2018-tcga-machine-learning.png
image-alt: >-
  Graphical abstract. Machine learning predicts Ras activation across 33 cancer types using genomic data, identifying
  phenocopying variants and MEK inhibition sensitivity.
members:
  - nils-gehlenborg
year: 2018
type: article
publisher: 'https://linkinghub.elsevier.com/retrieve/pii/S2211124718303899'
doi: 10.1016/j.celrep.2018.03.046
cite:
  authors: "*The Cancer Genome Atlas Research Network* (incl. N Gehlenborg)"
  published: '*Cell Reports* **23**(1):172-180.e3'
zotero-key: IWMTJ29F
videos: []
other-resources: []
awards: []
---
Precision oncology uses genomic evidence to match patients with treatment but
often fails to identify all patients who may respond. The transcriptome of
these “hidden responders” may reveal responsive molecular states. We describe
and evaluate a machine-learning approach to classify aberrant pathway activity
in tumors, which may aid in hidden responder identification. The algorithm
integrates RNA-seq, copy number, and mutations from 33 different cancer types
across The Cancer Genome Atlas (TCGA) PanCanAtlas project to predict aberrant
molecular states in tumors. Applied to the Ras pathway, the method detects Ras
activation across cancer types and identifies phenocopying variants. The model,
trained on human tumors, can predict response to MEK inhibitors in wild-type
Ras cell lines. We also present data that suggest that multiple hits in the Ras
pathway confer increased Ras activity. The transcriptome is underused in
precision oncology and, combined with machine learning, can aid in the
identification of hidden responders.
